sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -159,11 +165,52 @@ public partial class RedisEnterprisePrivateEndpointConnectionResource : Azure.Re
 {
+    public partial class Capability
+    {
+        internal Capability() { }
+        public string Name { get { throw null; } }
+        public bool? Value { get { throw null; } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryption
+    {
+        public ClusterPropertiesEncryptionCustomerManagedKeyEncryption() { }
+        public Azure.ResourceManager.RedisEnterprise.Models.ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity KeyEncryptionKeyIdentity { get { throw null; } set { } }
+        public System.Uri KeyEncryptionKeyUri { get { throw null; } set { } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity
+    {
+        public ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity() { }
+        public Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType? IdentityType { get { throw null; } set { } }
+        public string UserAssignedIdentityResourceId { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CmkIdentityType : System.IEquatable<Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public CmkIdentityType(string value) { throw null; }
+        public static Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType SystemAssignedIdentity { get { throw null; } }
+        public static Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType UserAssignedIdentity { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType left, Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType left, Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
     public partial class ExportRedisEnterpriseDatabaseContent
     {
         public ExportRedisEnterpriseDatabaseContent(System.Uri sasUri) { }
         public System.Uri SasUri { get { throw null; } }
     }
+    public partial class FlushContent
```

Comment:
Could we rename this class to `RedisEnterpriseDatabaseResourceFlushContent`? Since this is only used there